aregion.cpp

```diff
@@ -1564,6 +1564,12 @@ void ARegion::write_text_report(ostream &f, Faction *fac, int month, ARegionList
 			f << temp.const_str() << "\n";
 		}
 		
+		// Freezing effect
+		if (weather == W_BLIZZARD && !clearskies) {
+			temp = "There was an unnatural freezing last month.";
```

Comment:
I will need to check before turn that it does not break clients

Reply:
It likely will if any clients are parsing weather